src/redux/actions/componentConfig/getTrajectoryPlotStartingNodes.js

```diff
@@ -18,41 +16,16 @@ const getTrajectoryPlotStartingNodes = (
 
   const {
     clusteringSettings,
-    embeddingSettings: { methodSettings, useSaved },
+    embeddingSettings: { methodSettings },
   } = getState().experimentSettings.processing.configureEmbedding;
 
-  const { environment } = getState().networkResources;
-
-  const {
-    pipeline:
-    { startDate: qcPipelineStartDate },
-  } = getBackendStatus(experimentId)(getState()).status;
-
-  const embeddingBody = {
-    name: 'GetEmbedding',
-    type: embeddingMethod,
-    config: methodSettings[embeddingMethod],
-    useSaved,
-  };
-
-  const embeddingETag = await generateETag(
```

Comment:
I am not sure, but check if this isn't being used in the same way as in my previous trajectory analysis comment. If that is the case I don't think it can be removed either